apidef/api_definitions.go

```diff
@@ -365,6 +373,7 @@ type ExtendedPathsSet struct {
 	TrackEndpoints          []TrackEndpointMeta   `bson:"track_endpoints" json:"track_endpoints,omitempty"`
 	DoNotTrackEndpoints     []TrackEndpointMeta   `bson:"do_not_track_endpoints" json:"do_not_track_endpoints,omitempty"`
 	ValidateJSON            []ValidatePathMeta    `bson:"validate_json" json:"validate_json,omitempty"`
```

Comment:
We removed s.fillValidateRequest(ep.ValidateJSON) below - this field is orphaned now? To remove?

Reply:
This is for old users we will handle it in migration step later